gradle/defaults-java.gradle

```diff
@@ -25,13 +25,13 @@ allprojects {
     tasks.withType(JavaCompile) {
       options.encoding = "UTF-8"
       options.compilerArgs += [
-        "-Xlint", 
+        "-Xlint",
         "-Xlint:-deprecation",
         "-Xlint:-serial",
 
         "-Xdoclint:all/protected",
-        "-Xdoclint:-missing", 
-        "-Xdoclint:-accessibility", 
+        "-Xdoclint:-missing",
```

Comment:
extra white spaces.

Reply:
I wish I knew how this keeps happening, saw that after the PR. I commented out the -deprecation. Then uncommented. Then saved it. Didn't change he other lines at all AFAIK. I can fix it again, and apparently never touch that file again.